plugin.video.mlbtv/resources/lib/utils.py

```diff
@@ -0,0 +1,74 @@
+import os, re, sys
+from kodi_six import xbmc, xbmcaddon
+
+if sys.version_info[0] > 2:
+    import http
+    cookielib = http.cookiejar
+else:
+    import cookielib
+
+
+class Util:
+    addon_path = xbmc.translatePath(xbmcaddon.Addon().getAddonInfo('profile'))
```

Comment:
Xbmcvfs for matrix ++

Reply:
In order to take advantage of cross compatibility between python 2 and 3 wouldn't it make more sense to keep this as is so that I only have to keep one master branch that can work on both versions kodi?
Same with using `str(settings.getSetting(id="username"))` instead of `settings.getSettingStrinng(id="username")`

Reply:
+/-, the problem here is that we warn in matrix if you use the xbmc method

Reply:
for translatePath, I've went with 
```
try:
    xbmc.translatePath = xbmcvfs.translatePath
except AttributeError:
    pass
```
after the imports to make xbmc.translatePath() compatible